pkg/apis/apiserver/organization_REST.go

```diff
@@ -185,11 +185,12 @@ func (o *OrganizationREST) Create(ctx context.Context, obj runtime.Object, creat
 	if err := createValidation(ctx, obj); err != nil {
 		return nil, err
 	}
-	visible, err := o.isVisible(ctx, org)
+	// here we're not using checkIsOwner since we're returning different error
+	isOwner, err := o.containsUser(ctx, org.Spec.Owners)
```

Comment:
I don't think this check is needed, we can just return `apiservers.Forbidden` later in the `checkIsOwner`, they are doing the same thing.

Reply:
No, the error message is important. Otherwise, the user would be dumbfolded what is going on. Imagine you're creating the organization and you get forbidden error. But you should be allowed to do so. It leads to confusion and bad user experience. 


Do you want supporting user tickets like that and explaining it why the error message is wrong?

Reply:
Hmm, now I don't really agree with the workflow anymore.
For better user experience, instead of returning `BadReuqest` error, why don't we set User to the Organization Owner directly and let user create the Organization directly?  User doesn't need to remember to add himself/herself to the owner list every time to create organizations. For example, I have never been asked to set myself as Project owner in the GCP.

Reply:
how do you add user to owners? Do you add the username? Do you add the group? Do you do something else? Explicit is better than implicit in this case, and I'd prefer explicit setup